### ISSUE TITLE
Support Bluetooth connections

### ISSUE THREAD

**Issue by crossan007** [meshtastic/meshtastic-web#19]
Support connecting to radios over bluetooth and HTTP.

This would be a great addition to the app when adding PWA Support - #18

**Comment by Black6spdZ** [comment]
I disagree, whole point of getting functioning web interface is to be able to use wifi connected devices, phones, pc's tablets the whole lot connected simultaneously if needed. BT on modern phones comes at the cost of massive privacy
